seccomp.go

```diff
@@ -45,7 +45,7 @@ func init() {
 	// This forces the cgo libseccomp to initialize its internal API support state,
 	// which is necessary on older versions of libseccomp in order to work
 	// correctly.
-	GetAPI()
+	_, _ = GetAPI()
```

Comment:
This patch is no longer needed due to commit https://github.com/seccomp/libseccomp-golang/commit/ef015460ecc91e963a187dd865453cacb6f31899

Reply:
Rebased to remove this.